config/initializers/audit_instrumentation.rb

```diff
@@ -0,0 +1,32 @@
+# Audit logging from fact importer events sent via ActiveSupport::Notifications
```

Comment:
It is not generic, so I don't expects other classes to use it. We already have similar initializer. It's explained here: http://api.rubyonrails.org/classes/ActiveSupport/Notifications.html I am not sure what you expect me to add.